exercises/twelve-days/src/test/java/TwelveDaysTest.java

```diff
@@ -22,16 +22,16 @@ public void testVerseOne() {
     @Ignore("Remove to run test")
     @Test
     public void testVerseTwo() {
-        String expectedVerseTwo = "On the second day of Christmas my true love gave to me, two Turtle Doves, " +
+        String expectedVerseTwo = "On the second day of Christmas my true love gave to me, " + "two Turtle Doves, " +
```

Comment:
These changes aren't needed. It doesn't matter where you split up the string. Normally you would only do it if it doesn't all fit on one line. Does that make sense? 🙂

Reply:
Yup. I'll fix that up. Just to clarify: split up strings only when it doesn't fit on one line or if it's over 120 characters?

Reply:
Splitting lines of code doesn't have a 'hard' rule, some prefer 120 characters, some prefer 80. Other people might even have more rules! [Here](https://stackoverflow.com/questions/3137778/how-and-where-to-break-long-lines-of-code) is a link that might go into some more info. As for here, I don't think there is a written rule around this. As long as it is sensible, it should be ok 🙂

Reply:
Alright, thanks @sjwarner-bp!